config/galaxy.yml.k8s_interactivetools

```diff
@@ -0,0 +1,20 @@
+galaxy:
+  conda_auto_init: false
+  enable_data_manager_user_view: true
+  enable_tool_document_cache: true
+  integrated_tool_panel_config: /galaxy/server/config/mutable/integrated_tool_panel.xml
+  interactivetools_enable: true
+  interactivetools_map: database/interactivetools_map.sqlite
+  interactivetools_prefix: "its"
```

Comment:
Can you describe this and` interactivetools_shorten_url` in `lib/galaxy/webapps/galaxy/config_schema.yml` and then run `make config-rebuild` ?
I'd also drop this file and reference the necessary options in `config/job_conf.yml.sample_advanced` or `config/job_conf.xml.sample_advanced`

Reply:
Moved to schema in https://github.com/galaxyproject/galaxy/pull/11947/commits/d0971cc1b5e7bd1fccf9d61f6128717b1f258f48.
Re the separate file, I was copying `job_conf.xml.interactivetools` and thought it'd be helpful to have a similar file for k8s ITs. It being YML is necessary for it to work with the chart as expected.

Reply:
I think that was an oversight when ITs were first merged. There's so many different ways you can configure Galaxy and Galaxy jobs that this isn't maintainable way to document settings. In this file for instance there's a bunch of unrelated settings that as someone looking for examples you first need to understand why they are set to the values they are set and there's no explanation around it either.